src/abstractdataframe/join.jl

```diff
@@ -247,6 +247,12 @@ Join two `DataFrame` objects
   if `true`, duplicate names will be suffixed with `_i`
   (`i` starting at 1 for the first duplicate).
 
+* `indicator` : Default: `nothing`. If a `Symbol`, adds categorical indicator
+   column for whether a row appeared in only `df1` ("left_only"),
+   only `df2` ("right_only") or in both ("both").
+   By default column will use the passed `Symbol` as its name unless
```

Comment:
I find this a bit confusing. Maybe better say "add categorical indicator column with the specified name" in the previous sentence. Then you can explain what happens in case of conflict: an error is raised, unless `makeunique` is `true`.

Reply:
So, re: #1425 , the problem is at the moment this doesn't raise an error. It ignores the command...